### PR TITLE
WRO-6381: Fix security vulnerabilities

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
For preventing malware issues, we need to update node modules that fix severity vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update dependencies to the latest versions.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Update the lockfile version to v2

### Links
[//]: # (Related issues, references)
WRO-6381

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,26 +39,26 @@
...
   "devDependencies": {
-    "@enact/docs-utils": "^0.3.6",
-    "@enact/ui-test-utils": "^1.0.0-alpha.1",
-    "eslint-config-enact-proxy": "^1.0.4",
+    "@enact/docs-utils": "enactjs/docs-utils#feature/WRO-6381",
+    "@enact/ui-test-utils": "^1.0.0",
+    "eslint-config-enact-proxy": "^1.0.5",
     "ilib": "^14.14.0"
   }
 }
```
Comment (MikyungKim) [inline package.json:64]: please add an empty line at the end.

**Review by MikyungKim — COMMENTED** [review]
Would you please update versions of the samples?

**Review by MikyungKim — COMMENTED** [review]
Please update enact packages to the latest.

**Comment by hong6316** [comment]
> Would you please update versions of the samples?

OK

**Comment by hong6316** [comment]
> Please update enact packages to the latest.

OK

File: samples/qa-virtuallistnative/package.json [hunk 2]
```diff
@@ -30,17 +30,17 @@
     "extends": "enact/strict"
   },
   "dependencies": {
-    "@enact/core": "^4.5.0-rc.1",
-    "@enact/i18n": "^4.5.0-rc.1",
+    "@enact/core": "^4.5.0-rc.2",
```
Comment (MikyungKim) [inline samples/qa-virtuallistnative/package.json]: You didn't update this package.json.

Comment (hong6316) [reply samples/qa-virtuallistnative/package.json]: I missed it, OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**